AWS/relayer.env

```diff
@@ -5,5 +5,9 @@ ROLLAPP_CHAIN_ID=rollappevm_100_1
 EXECUTABLE=rollapp_evm
 SETTLEMENT_RPC_FOR_RELAYER="ec2-18-202-218-150.eu-west-1.compute.amazonaws.com:36657"
 ROLLAPP_RPC_FOR_RELAYER="ec2-34-249-17-158.eu-west-1.compute.amazonaws.com:26667"
-RELAYER_PATH="hub-rollapp"
+
+# This path establishes a relayer channel from the rollapp to the hub. 
+RELAYER_PATH="hub-rollapp" 
+
+# This channel will be generated by the setup_ibc.sh script. Make sure to replace the value with the generated source chain channel name.
```

Comment:
not very clear to me.
Does this value sets the channel name or is it determined by the setup_ibc and later it needs to be udpated?
Also you could specify how to get the channel..